File: SDKs/twilio.com/twilio_voice_v1/1.36.0/go/pkg/models/operations/fetchconnectionpolicy.go
package operations

import (
"openapi/pkg/models/shared")
var FetchConnectionPolicyServers = []string{
	"https://voice.twilio.com",
}

type FetchConnectionPolicyPathParams struct {
    Sid string `pathParam:"style=simple,explode=false,name=Sid"`
    
}

type FetchConnectionPolicySecurity struct {
    AccountSidAuthToken shared.SchemeAccountSidAuthToken `security:"scheme,type=http,subtype=basic"`
    
}

type FetchConnectionPolicyRequest struct {
    ServerURL *string 
    PathParams FetchConnectionPolicyPathParams 
    Security FetchConnectionPolicySecurity 
    
}

type FetchConnectionPolicyResponse struct {
    ContentType string 
    StatusCode int64 
    VoiceV1ConnectionPolicy *shared.VoiceV1ConnectionPolicy 
    
}

File: SDKs/twilio.com/twilio_voice_v1/1.36.0/typescript/src/sdk/models/operations/fetchconnectionpolicy.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";
export const FETCHCONNECTIONPOLICY_SERVERS = [
	"https://voice.twilio.com",
];



export class FetchConnectionPolicyPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=Sid" })
  sid: string;
}


export class FetchConnectionPolicySecurity extends SpeakeasyBase {
  @Metadata({ data: "security, scheme=true;type=http;subtype=basic" })
  accountSidAuthToken: shared.SchemeAccountSidAuthToken;
}


export class FetchConnectionPolicyRequest extends SpeakeasyBase {
  @Metadata()
  serverUrl?: string;

  @Metadata()
  pathParams: FetchConnectionPolicyPathParams;

  @Metadata()
  security: FetchConnectionPolicySecurity;
}


export class FetchConnectionPolicyResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;

  @Metadata()
  voiceV1ConnectionPolicy?: shared.VoiceV1ConnectionPolicy;
}
